### DISCUSSION TITLE
Disabling TLS on 0.14.0-beta2 generates error message every minute

### DISCUSSION THREAD

**Discussion by themoosman** [blakeblackshear/frigate#11683] [General]
When disabling TLS via a blank tls.conf file, Frigate generates the following error every minute.

```
unable to load certificate
140249079821632:error:0909006C:PEM routines:get_name:no start line:../crypto/pem/pem_lib.c:745:Expecting: TRUSTED CERTIFICATE
```

This also causes the nginx config to be reloaded, which is unnecessary.  

In my case, I don't need Frigate to do anything with TLS, as there is a reverse proxy that handles SSL.

Also, why is the new image trying to listen on a privileged port (443)?

**Comment by sneakrz** [comment]
I could no longer use the host networking option, since I'm also using a reverse proxy. Then I only exposed ports 5000, 8080, 8554, and 8555. Problem solved, hopefully. I didn't have to do anything with a tls.conf file.

Edit: Ports are listed on [this](https://deploy-preview-11419--frigate-docs.netlify.app/frigate/installation#ports) page.

**Reply by themoosman** [reply]
If you supply a blank tls.conf file, it won't listen on 443.  This seems to make everything work like beta1.

**Comment by blakeblackshear** [comment]
Decided to make some significant changes to the TLS implementation that should address all of this: https://github.com/blakeblackshear/frigate/pull/11690

**Reply by themoosman** [reply]
I just tested the dev build, and it works great.  Thanks.
